src/utils/tls.rs

```diff
@@ -96,7 +96,7 @@ fn read_key(key_path: &Path) -> Result<PrivateKey, ExitError> {
         })?;
 
         let bits = match item {
-            RSAKey(bits) | PKCS8Key(bits) => bits,
+            RSAKey(bits) | PKCS8Key(bits) | ECKey(bits) => bits,
```

Comment:
Is this tested anywhere?

Reply:
Not currently, no.

Reply:
I tested it locally like so:

```
$ openssl ecparam -name prime256v1 -genkey -noout -out /tmp/ec-private-key.pem
$ openssl ec -in /tmp/ec-private-key.pem -pubout -out /tmp/ec-public-key.pem
$ openssl req -new -x509 -key /tmp/ec-private-key.pem -out /tmp/ec-cert.pem -days 360
$ routinator -vvv server --http-tls 127.0.0.1:8443 --http-tls-cert /tmp/ec-cert.pem --http-tls-key /tmp/ec-private-key.pem 
```

With Routinator 0.13.0 that resulted in an error:

```
[ERROR] TLS key file '/tmp/ec-private-key.pem' does not contain any usable keys.
```

With this PR there was no error and `openssl` shows the EC key in use:

```
$ true | openssl s_client -connect localhost:8443 2>&1 | openssl x509 -noout -text | grep Algorithm
        Signature Algorithm: ecdsa-with-SHA256
            Public Key Algorithm: id-ecPublicKey
    Signature Algorithm: ecdsa-with-SHA256
```